doc/classes/JavaObject.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="JavaObject" inherits="RefCounted" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		Represents an object from the Java Native Interface.
+	</brief_description>
+	<description>
+		Represents an object from the Java Native Interface. It can be return from Java methods called on [JavaClass]. See [JavaClassWrapper] for an example.
```

Comment:
nit: typo on `return`; should be `returned`

Reply:
Thanks! Updated in my latest push